### DISCUSSION TITLE
开源治理系列参考规范的总体框架

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#27] [00 开源治理系列参考规范整体设计]
这个总体框架中有三大块内容：

- 开源治理系列参考标准
- 基础设施与工具
- 实践与应用

<img src="https://user-images.githubusercontent.com/15010826/158111929-ba27e324-3e98-411d-b956-88c07fbde2aa.png" width="400px">

（1）开源治理系列参考标准

参考标准体系中有五大关键模块：

- 开源术语：[讨论区](https://github.com/kaiyuanshe/ONES/discussions/categories/01-%E5%BC%80%E6%BA%90%E6%9C%AF%E8%AF%AD)
- 开源元数据：[讨论区](https://github.com/kaiyuanshe/ONES/discussions/categories/02-%E5%BC%80%E6%BA%90%E5%85%83%E6%95%B0%E6%8D%AE)
- 企业开源治理：[讨论区](https://github.com/kaiyuanshe/ONES/discussions/categories/03-%E4%BC%81%E4%B8%9A%E5%BC%80%E6%BA%90%E6%B2%BB%E7%90%86)
- 社区治理与运营：[讨论区](https://github.com/kaiyuanshe/ONES/discussions/categories/04-%E5%BC%80%E6%BA%90%E7%A4%BE%E5%8C%BA%E6%B2%BB%E7%90%86%E4%B8%8E%E8%BF%90%E8%90%A5)

（2）基础设施与工具

与开源开源系列参考标准对应的工具、数据、基础设施、服务等，例如数据工具、合规工具、代码成分分析工具、度量工具等等。

（3）实践与应用

包括与系列规范对应的企业最佳开源治理实践、行业开源治理案例、开源治理工具与服务评价、开源治理培训课程与咨询服务等。

**Comment by will-ww** [comment]
更新一下整体框架图：

<img src="https://user-images.githubusercontent.com/15010826/161353647-fe383a66-7f0c-45dc-9ac9-d94a926673db.png" width="400px">

**Comment by wangleo61** [comment]
这个图非常好呀

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Will ***@***.***>
发送时间: Saturday, April 2, 2022 7:21 AM
收件人: ***@***.***>
抄送: ***@***.***>
主题: Re: [kaiyuanshe/ONES] 开源治理系列参考规范的总体框架 (Discussion #27)


更新一下整体框架图：

[image]<https://user-images.githubusercontent.com/15010826/161353647-fe383a66-7f0c-45dc-9ac9-d94a926673db.png>

―

**Reply by will-ww** [reply]
接下来，我们就开始分工协作了，一起一起呀~ @wangleo61

**Comment by wangleo61** [comment]
太好啦，期待期待

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Will ***@***.***>
发送时间: Sunday, April 3, 2022 7:17 AM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] 开源治理系列参考规范的总体框架 (Discussion #27)


接下来，我们就开始分工协作了，一起一起呀~ @wangleo61<https://github.com/wangleo61>

―
